src/Administration/Resources/administration/src/app/component/form/select/base/sw-multi-select/index.js

```diff
@@ -38,7 +38,7 @@ Component.register('sw-multi-select', {
         },
         value: {
             type: Array,
-            required: true
+            required: false
```

Comment:
this is failing because with value = undefined the isSelected Method throws an error. 
```
    methods: {
        isSelected(item) {
            return this.currentValue.includes(this.getKey(item, this.valueProperty));
        },
```
This should work:
```
        value: {
            type: Array,
            required: false,
            default() {
                return [];
            }
        },
```